main.go

```diff
@@ -187,7 +228,7 @@ func collectStatsForWithError(proc *process.Process, withError bool) (*MozProces
 			fmt.Printf("MemoryInfo: %s\n", err)
 		}
 	} else {
-		statistics.Memory = ProcMemoryInfoStat{memory.RSS, memory.VMS, memory.Swap, 0}
```

Comment:
This makes sense, I was originally confused by this. We skipped it during adding and then inject `0` value. I think it's much clearer now with dedicated value at line #296.